packages/cspell-eslint-plugin/package.json

```diff
@@ -80,6 +80,7 @@
     "@eslint/eslintrc": "^3.1.0",
     "@eslint/js": "^9.5.0",
     "@internal/cspell-eslint-plugin-scripts": "workspace:*",
+    "@isentinel/dict-roblox": "^1.0.0",
```

Comment:
Do you need to depend on it? Or is it only for the test?

I assumed you added example to simulate it was working.

Please note, I have no idea what I'm talking 😅

But I'm raising the point anyway as I'm surprised

Reply:
It is just for tests.

Related to #5787, I just used his example.

Reply:
I saw the ticket earlier. I missed the fact it was in `devDependencies` node

Reply:
But, I'll make a local dictionary to use for testing. I don't like to add unnecessary dependencies.

Thank you for raising the issue.

Reply:
My pleasure